pymeasure/instruments/toptica/ibeamsmart.py

```diff
@@ -227,7 +227,7 @@ def check_errors(self):
         get_process=lambda s: True if s == 'ON' else False,
         set_process=lambda v: "on" if v else "off",
         check_set_errors=True,
-        command_process=_deprecation_warning(
+        preprocess_reply=_deprecation_warning(
```

Comment:
I assume there's a test present already that confirms that that deprecation warning is still being emitted?

Reply:
No, there was no test.
I added a test and modified the code to warn also on setting (which I forgot)